tests/e2e-tests/pom.xml

```diff
@@ -75,6 +101,51 @@
 
    <build>
       <plugins>
+         <plugin>
+            <!-- TODO: Review Codehaus usage...ARTEMIS-3750 -->
+            <groupId>org.codehaus.mojo</groupId>
+            <artifactId>exec-maven-plugin</artifactId>
+            <version>3.0.0</version>
```

Comment:
Should be in a property, and the version controlled in pluginManagement at the root so it is consistent across the build and can be updated easily.

Reply:
PR updated